tests/phpunit/tests/test-slugs.php

```diff
@@ -121,4 +121,53 @@ public function test_update_existing_term_slugs_with_parent() {
 		$this->assertSame( 'New Test', $en_new->name );
 		$this->assertSame( 'test-en', $en_new->slug );
 	}
+
+	public function test_untranslatable_taxonomy() {
+		register_taxonomy(
+			'test-tax',
+			'post',
+			array(
+				'public' => false, // Not translatable by default.
+			)
+		);
```

Comment:
In fact all taxonomies and post types are not translatable by default.
```suggestion
		register_taxonomy( 'test-tax', 'post' ); // Not translatable by default.
```